ajax/libs/PgwSlider/package.json

```diff
@@ -0,0 +1,34 @@
+{  
+  "name":"PgwSlider",
+  "npmName":"pgwslider",
+  "npmFileMap":[  
+    {  
+      "basePath":"",
+      "files":[  
+        "*"
```

Comment:
Let's use git auto-update because there are more versions on git repo.
It's like
```js
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/Pagawa/PgwSlider.git",
    "fileMap": [
      {
        "basePath": "",
        "files": [
          "pgwslider*.+(css|js|map)"
        ]
      }
    ]
  }
```
I changed the file pattern to `pgwslider*.+(css|js|map)`  because `*` will grab all file in root directory. The current one can grab only what CDN needs.